src/magic_spell_effect.cpp

```diff
@@ -1511,6 +1511,11 @@ void spell_effect::effect_on_condition( const spell &sp, Creature &caster, const
         }
         dialogue d( get_talker_for( g->critter_at<Creature>( potential_target ) ),
                     get_talker_for( caster ) );
-        effect_on_condition_id( sp.effect_data() )->activate( d );
+        effect_on_condition_id eoc = effect_on_condition_id( sp.effect_data() );
+        if( eoc->activate_only ) {
+            eoc->activate( d );
+        } else {
+            debugmsg( "Cannot use a recurring effect_on_condition in a spell.  If you don't wish the effect_on_condition to happen on its own remove the recurrence min and max.  Otherwise create a non recurring effect_on_condition for this spell with its condition and effects and have a recurring one queue it." );
```

Comment:
```suggestion
            debugmsg( "Cannot use a recurring effect_on_condition in a spell.  If you don't want the effect_on_condition to happen on its own, remove the recurrence min and max.  Otherwise, create a non-recurring effect_on_condition for this spell with its condition and effects, then have a recurring one queue it." );
```